benchmarks/include/Config.h

```diff
@@ -0,0 +1,106 @@
+#pragma once
```

Comment:
can we follow LLVM style here?

Reply:
I don't think we need to, because this will never touch LLVM code, we'll never want to upstream this code, and `#pragma once` is superior.

With the `ifndef` trick, it's all too common to copy and paste and forget to change, and the compiler won't complain until you include both headers on the same file.